pheps/phep-0002.md

```diff
@@ -0,0 +1,127 @@
+```
+PHEP: 2
+Title: PHEP Template
+Author: Jonathan T. Niehof <jtniehof@gmail.com> <https://orcid.org/0000-0001-6286-5809>
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/25
+Revision: 1
+Status: Draft
+Type: Informational
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Requires: 1
+Created: 06-Dec-2023
+Post-History: 06-Dec-2023, 28-Feb-2024, 20-Aug-2024
+Resolution: https://docs.google.com/document/d/12IGirV5RM50LqhifXBE_UzY10LDuoVxaS5bG6QWlL4M/edit
+```
+
+# Abstract
+<a name="abstract"></a>
+This PHEP provides a template to make it easier to write a PHEP-1 compliant PHEP. It does not establish or enforce any standard. Each section contains explanatory text for how to write that section in an actual PHEP.
+
+The majority of the text should be replaced with the text of your proposed PHEP. See [Specification](#specification) for further directions on using this template.
+
+This section of your PHEP should include a short description of the issue being addressed.
+
+# Motivation
+<a name="motivation"></a>
+Discussions in the drafing of PHEP 1 indicated a desire to have a template to make it easier for potential PHEP writers to comply with the format, address the required elements, and avoid the "blank page" problem.
+
+The motivation must clearly explain why the existing standards are inadequate to address the problem that the PHEP solves. This can include collecting documented support for the PHEP from important projects in the PyHC ecosystem.
+
+# Rationale
+<a name="rationale"></a>
+The template model should help make the initial drafting of a PHEP a fairly simple copy-paste-replace operation. Although in theory any PHEP could be taken as a template, each PHEP is unique and it is easier to have one that spans the set of expected sections, while minimizing additional information.
+
+The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work.
+
+The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during discussion.
+
```

Comment:
```suggestion

# Use cases
<a name="use-cases"></a>

```
I'm wondering — would it make sense to have a section in the template for [use cases](https://en.wikipedia.org/wiki/Use_case#:~:text=In%20software%20and%20systems%20engineering,%20the%20phrase%20use%20case%20is)?

Reply:
That _might_ go in "how to teach this" although I also understand how it would make sense in rationale.

The structure of PHEP2 directly follows the sections from PHEP1. So structural changes should probably go in #27. Further guidance/suggestion/nudges could go in here. I'm a little nervous about having substantive guidance hiding within a template though--in theory PHEP2 is merely a convenience not some sort of definitive, or even best-practices, reference.